### DISCUSSION TITLE
The column "analysis_type" in file.tsv

### DISCUSSION THREAD

**Discussion by mano-at-sdsc** [nih-cfde/published-documentation#340] [Q&A] [answered]
I assume "analysis_type" refers to computational analysis type here. The example given there for analysis_type [https://github.com/nih-cfde/published-documentation/wiki/TableInfo:-file.tsv] is for assay_type. Please provide a suitable example and list the website or resource where we can search for analysis_type [e.g., in an {external} reference table. Thanks.

**Comment by ACharbonneau** [comment, marked as answer]
Hi there, the example you are citing is correct. Both analysis type and assay type use the same OBI CV terms.

**Comment by mano-at-sdsc** [comment]
Thanks, Amanda. In that case, I will just use the same values in both assay_type column and analysis_type column.

Mano
